lib/rssbridge.php

```diff
@@ -74,6 +74,7 @@
 require_once PATH_LIB . 'ParameterValidator.php';
 require_once PATH_LIB . 'ActionFactory.php';
 require_once PATH_LIB . 'ActionAbstract.php';
+require_once PATH_LIB . 'UserDefinedAbstract.php';
```

Comment:
We don't have to manually require classes due to autoloading. The other ones are leftovers.